todo_app/data/trello_items.py

```diff
@@ -0,0 +1,136 @@
+import requests
```

Comment:
How did you install requests?
The correct thing to do is `poetry add requests`, which should update pyproject.toml but that change hasn't been included in this PR.
You need that change otherwise `poetry install` won't be enough for other people to run the app.